packages/message/src/Collections.ts

```diff
@@ -25,6 +29,53 @@ import { Message } from '@cumulus/types';
 export const constructCollectionId = (name: string, version: string) =>
   `${name}___${version}`;
 
+/**
+ * Get collection name from execution message.
+ *
+ * @param {Message.CumulusMessage} message - An execution message
+ * @returns {string | undefined} - Collection name or undefined
+ *
+ * @alias module:Collections
+ */
+export const getCollectionNameFromMessage = (
+  message: Message.CumulusMessage
+): string | undefined => message.meta?.collection?.name;
+
+/**
+ * Get collection version from execution message.
+ *
+ * @param {Message.CumulusMessage} message - An execution message
+ * @returns {string | undefined} - Collection version or undefined
+ *
+ * @alias module:Collections
+ */
+export const getCollectionVersionFromMessage = (
+  message: Message.CumulusMessage
+): string | undefined => message.meta?.collection?.version;
+
+/**
+ * Get collection name and version from execution message.
+ *
+ * @param {Message.CumulusMessage} message - An execution message
+ * @returns {CollectionInfo | undefined}
+ *   Object with collection name and version or undefined
+ *
+ * @alias module:Collections
+ */
+export const getCollectionInfoFromMessage = (
```

Comment:
Nit (non blocking): it's obnoxious, but getCollectionNameAndVersionFromMessage would be explicit re: what 'info' is, for readability.     Too bad there isn't a good accepted name for a collection name/version object.   I suppose we could type it and reference the type, but that's probably overkill.